cohortextractor/emis_backend.py

```diff
@@ -70,7 +71,7 @@ def to_dicts(self):
         output = [dict(zip(keys, map(str, row))) for row in result]
         unique_check = UniqueCheck()
         for item in output:
-            unique_check.add(item["patient_id"])
+            unique_check.add(item["registration_id"])
```

Comment:
There may well be good reasons for doing this, but as an initial reaction the fact that EMIS uses `registration_id` internally feels like a bit of an implementation detail that shouldn't leak out here.

Reply:
Have reverted in 45f442f2aafa23451399b6c6ef46cacfd2ff2885.